modules/templates/helper.go

```diff
@@ -523,6 +544,38 @@ func SVG(icon string, others ...interface{}) template.HTML {
 	return template.HTML("")
 }
 
+// Avatar renders user and repo avatars. args: user/repo, size (int), class (string)
+func Avatar(item interface{}, others ...interface{}) template.HTML {
+	size, class := parseOthers(28, "ui avatar image", others...)
+	if user, ok := item.(*models.User); ok {
+		src := user.RealSizedAvatarLink(size * 2) // request double size for finer rendering
+		if src != "" {
+			return avatarHTML(src, size, class, user.DisplayName())
+		}
+	}
+
```

Comment:
if it's `git.Signature`??

Reply:
You mean typecheck for that as well? Where is it used? Should we use it in place of email?